app/src/main/res/layout/modal_bottom_sheet_dialog.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:id="@+id/download"

```

Comment:
try to keep coherency in names. Either rename shareLinearLayout -> share and uploadLinearLayout -> upload or rename download -> downloadLinearLayout